### PR TITLE
MNG-96 pokemon attack

### PR DESCRIPTION
**Pull request by daria305**

### CODE REVIEW

**Review by AleksandraCyp — CHANGES_REQUESTED** [review]
Hi Daria, I already created a system that checkes if the pokemon was defeated and forces you to choose a new pokemon and on my current branch, with a lot less lines of code. After it gets merged, please check it out!

Basically, I think that the code you added in the attack function should be enough. You don't have to change anything in buttons!

The only thing regarding buttons that is to be made is desabling them when a given pokemon is dead (but there is a task on trello for that, I think)

File: src/fightPage/buttonsEventListeners.ts
Comment (AleksandraCyp) [inline src/fightPage/buttonsEventListeners.ts]:
```
const pokemon = gameHandler.currentPlayer.getActivePokemon;
const move = pokemon.moves.find(move => move.moveName === e.currentTarget.textContent)
```
Maybe instead of adding moves to magic funtion and event listeners, you could use the following code in attack function? (pass event as parameter to the attack function?)

Comment (daria305) [reply src/fightPage/buttonsEventListeners.ts]: ok, I used getting move by event.
I will merge it after you pull request will be on master.

**Review by AleksandraCyp — APPROVED** [review]

**Pull request merged**
